docs/docs/external/getting-started/enabling-weaver-network/fabric.md

```diff
@@ -0,0 +1,398 @@
+---
+id: fabric
+title: Enabling Weaver in a Fabric Network and Application
```

Comment:
Can this Title be short? In rendered page, the sub topics seems to be repetitive in information, here:
<img width="289" alt="image" src="https://user-images.githubusercontent.com/17192099/129321492-ed32c85a-29a7-441a-b74c-885967acfd1a.png">
I'd suggest to keep them as Fabric, Corda and Besu only?

Reply:
Just found out:
---
id: fabric
title: Enabling Weaver in a Fabric Network and Application
sidebar_label: Fabric
---

This can do what I was suggesting.
Can we use this for all pages in Testing Scenarios as  well? They also seemed repetitive.

Reply:
Agreed. Fixed.